specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2024-05-01-preview/appplatform.json

```diff
@@ -15925,12 +15914,42 @@
         "value"
       ]
     },
-    "EnvSecrets": {
-      "description": "Sensitive properties for environment variables",
+    "EnvVar": {
+      "description": "Azure Spring Apps components' environment variable.",
       "type": "object",
-      "x-ms-secret": true,
-      "additionalProperties": {
-        "type": "string"
+      "properties": {
+        "name": {
+          "description": "Environment variable name.",
+          "type": "string"
+        },
+        "value": {
+          "description": "Non-secret environment variable value.",
+          "type": "string"
+        },
+        "secretValue": {
+          "description": "secret environment variable value.",
+          "type": "string",
+          "x-ms-secret": true
+        }
+      }
+    },
+    "EnvSecretsCollection": {
+      "description": "Secret environment variable collection.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Secret"
```

Comment:
Where's this defined?

Reply:
It's using existing model [Secret](https://github.com/Azure/azure-rest-api-specs/blob/4c3a58fe52262b54e2cb334481820a975be715b0/specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2024-05-01-preview/appplatform.json#L12844).